pkg/formatted/version_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2021 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package formatted
+
+import (
+	"testing"
+
+	"github.com/tektoncd/cli/pkg/test"
+)
+
+func TestVersion(t *testing.T) {
+	version := make(map[string]string)
+
+	version["app.kubernetes.io/version"] = "v1"
```

Comment:
can we add one more test with other label to check whether it returns an empty string or not? eg `tekton.dev/name: foobar`